.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -221,6 +267,7 @@ jobs:
             mkdir -p cilium-junits
 
       - name: Wait for images to be available
+        if: ${{ matrix.mode != 'patch' }}
```

Comment:
These are the images for the PR under test, aren't they? Why don't we need those images when testing the patch up/downgrade path?

Reply:
Sounds right, I got confused and thought they were the ones for the patch releases. The tests still passed, likely because the image is usually ready in time, which comforted me in that belief... I'll revert this bit. Thanks!

Reply:
I'll propagate the fix to the backport PRs, too.